src/System.Runtime/tests/System/DateTime.cs

```diff
@@ -381,6 +381,23 @@ public static void TestSpecialCulturesParsing(string cultureName)
         Assert.Equal(date.TimeOfDay, parsedDate.TimeOfDay);
     }
 
+    /// <summary>
+    /// Since .NET Core doesn't expose DateTimeFormatInfo DateSeparator and TimeSeparator properties,
```

Comment:
The reason we didn't expose the date/time separators is because it is not right concept and people ran into problem when customizing dates. for example if some one has a date format like "dd MM/yyyy" what should be the date separator here? there is some real scenarios like people use dates in their language which is equivalent to the following in English "'Day' dd 'Month' MM 'Year' yyyy". the same problem apply to the time too.